common/src/main/java/com/wynntils/models/containers/BankModel.java

```diff
@@ -157,12 +222,13 @@ public void toggleEditingName(boolean editingName) {
         this.editingName = editingName;
     }
 
-    private Storage<Map<Integer, String>> getCurrentNameMap() {
+    private Map<Integer, String> getCurrentNameMap() {
```

Comment:
To be honest this change added more code than it needed to. Do you think this should be kept?

Reply:
It could be, would just need a special case for when it's the character bank

Reply:
Okay, I guess let it be for now.